cmd/scollector/collectors/snmp_sys.go

```diff
@@ -0,0 +1,34 @@
+package collectors
+
+import (
+	"fmt"
+	"math/big"
+	"time"
+
+	"bosun.org/cmd/scollector/conf"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+const sysUpTime = ".1.3.6.1.2.1.1.3.0" // "The time (in hundredths of a second) since the network management portion of the system was last re-initialized."
+
+// SNMPSys registers a SNMP system data collector for the given community and host.
+func SNMPSys(cfg conf.SNMP) {
+	collectors = append(collectors, &IntervalCollector{
+		F: func() (opentsdb.MultiDataPoint, error) {
+			return c_snmp_sys(cfg.Community, cfg.Host)
+		},
+		Interval: time.Minute * 1,
+		name:     fmt.Sprintf("snmp-sys-%s", cfg.Host),
+	})
+}
+
+func c_snmp_sys(community, host string) (opentsdb.MultiDataPoint, error) {
```

Comment:
You could probably do this by building a hard-coded MIB struct like the cisco one does. I'm ok with this though. Do what feels good. LGTM